docs/CONTRIBUTING.md

```diff
@@ -40,7 +40,7 @@ as a guide for writing proposals.
 2. Copy the proposal template and place it
    under `docs/proposals/<component_name>/README.md`
 3. Fill out the proposal template
-4. Submit a PR for the propsoal
+4. Submit a PR for the proposal
```

Comment:
You can drop this typo fix if you want, I have a whole PR up changing the contributing guidelines 😂

Reply:
Dropped the changes against this file. Thanks for mentioning this!